okhttp-tls/src/main/java/okhttp3/tls/HandshakeCertificates.java

```diff
@@ -137,6 +140,24 @@ public Builder addTrustedCertificate(X509Certificate certificate) {
       return this;
     }
 
+    /**
+     * Add all of the host platform's trusted root certificates. This set varies by platform
+     * (Android vs. Java), by platform release (Android 4.4 vs. Android 9), and with user
+     * customizations.
+     *
+     * <p>Most TLS clients should that connect to hosts on the public Internet should call this
```

Comment:
_Most TLS clients that connect_ ?

Reply:
Fixed. I have much difficulty copy-editing my own work, especially when I write and rewrite this stuff many times before I’m happy with it.